app/pages/delete-order/template.njk

```diff
@@ -0,0 +1,43 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/bc-button/macro.njk' import bcButton %}
+
+{% block body %}
+  <div class="nhsuk-width-container" data-test-id="delete-order-page">
+    <div data-test-id="go-back-link">
+      {{ backLink({
+        "href": backLinkHref,
+        "text": backLinkText
+      }) }}
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 data-test-id="delete-order-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
+        <h2 data-test-id="delete-order-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+        <h3 data-test-id="order-description-title" class="nhsuk-u-margin-bottom-3">{{ orderDescriptionTitle }}</h3>
+        <h4 data-test-id="order-description" class="nhsuk-u-font-size-19 nhsuk-u-font-weight-normal">{{ orderDescription }}</h4>
+
+        <form method="post">
+          <div class="nhsuk-grid-row">
+              <div class="nhsuk-grid-column-two-thirds">
+                <div class="bc-u-inline-block">
```

Comment:
This will probably be part of another story, so please ignore if that's the case & excuse my ignorance but when it comes to the post how will you know which button was pressed?

Reply:
This is part of another task, will talk with fawad when we start it

Reply:
*Just spoke to Fawad. The no button we are going to treat as a href, so then will just do the post on the yes button